understanding/20/language-of-parts.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Language of Parts</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Assistive technology can identify the languages used within a page</dd>
-		<dt>Author task</dt><dd>Indicate when words are in a different language</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Assistive technology can identify the languages used within a page.</dd>
+         <dt>What to do</dt><dd>Indicate when words are in a different language.</dd>
+         <dt>Why it's important</dt><dd>People using assistive technology get information in the correct language.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>People using assistive technology get information in the correct language and synthesizers will pronounce text properly.</dd> 
```

Reply:
@GreggVan also suggested
```suggestion
         <dt>Why it's important</dt><dd>People using assistive technology get information in the correct language, and speech synthesizers read the text with the right synthesizer and cognitive aids know what language the text is in.</dd> 
```

Reply:
How about
```suggestion
         <dt>Why it's important</dt><dd>People using assistive technology get context on the language, such as pronunciation.</dd> 
```

Reply:
In fact, I think context on a language and pronunciation is something different.
Context about French for example is, where it is spoken and how it evolved. Context about French pronunciation includes information how French words are pronounced.
This is not what the user needs. The user wants the information in an understandable form which implies the correct language.
If assistive technology identifies the language, it does not give additional informaiton to the end user on the language. It reads out the text correctly.
Therefore I suggest to keep the original wording.

Reply:
I take both your and Gregg's points. Sometimes screen readers will simply say "Chinese" (or whatever language) to identify when they encounter text in a language they do not support. But other times, where the synthesizer supports that language, the phrase will be read in the appropriate synthesized tongue -- in other words, pronounced.
This is most noticeable encountering words that are in similar language families where a series of characters may be pronounced quite differently. The synthesizer will **pronounce** it in French, instead of English, for example.

On the other hand, specified language is not going to help with regional pronunciation, as far as I know. Nor is it going to provide context for what the words mean.

How about 
```suggestion
         <dt>Why it's important</dt><dd>People using assistive technology get information about the languages in the  content.</dd> 
```

Reply:
In fact, I still prefer the original wording "People using assistive technology get information in the correct language."
Because this is what happens if the author indicated the correct language and the assistive technology received the language indication (tat is, when the author action has been completed and the goal is reached): the gets the information in the correct language.
Yes, provided the synthesizer supports the language , yet still the correct pronunciation is the motivation.

The user does not receive information about the langue (like history or so). The user might receive the information which language is is if the synthesizer does not support it, yet this is the behavior of the synthesizer in an error case and not the motivation to indicate the language.